src/extension/src/controller.ts

```diff
@@ -167,6 +171,31 @@ export class Controller {
     }
   }
 
+  private static getProjectName() {
+    const userOutputPath = vscode.workspace
+      .getConfiguration()
+      .get<string>("wts.changeSaveToLocation");
+    const outputPath: string = userOutputPath ? userOutputPath : os.homedir();
+    const defaultAppName = "myApp";
+    let newAppName = "myApp";
+    let count = 1;
+
+    while (
+      fs.existsSync(path.join(outputPath, newAppName)) &&
+      count <= PROJECT_NAME_VALIDATION_LIMIT
+    ) {
+      newAppName = `${defaultAppName}${count}`;
+      count++;
+    }
+    if (count > PROJECT_NAME_VALIDATION_LIMIT) return;
```

Comment:
Nit: This return is hard to notice at first glance. Can you move it to the next line at least? (Don't have to change if you feel the current way is better).

Reply:
Yeah, I will make it more noticeable :)